blocks/carousel/carousel.css

```diff
@@ -178,10 +189,21 @@ main .section div.carousel-wrapper {
     border-radius: 50%;
 }
 
+.carousel.case-study .carousel-dots li {
+    width: 68px;
+    height: 5px;
+    border-radius: unset;
+}
+
 .carousel .carousel-dots li.carousel-dots-active {
     box-shadow: inset 0 0 100px 100px var(--primary-purple);
 }
 
+.carousel.case-study .carousel-dots li.carousel-dots-active {
+    box-shadow: inset 0 0 100px 100px var(--gradient-left-right);
+    background-image: var(--gradient-left-right);
```

Comment:
You don't need to set the box-shadow property to get the active gradient. Just use the background-position (see attached).

```suggestion
    box-shadow: inherit;
    background-image: var(--gradient-left-right);
    background-position: 0 100%;
```

![Screenshot 2023-08-30 at 12 01 05 PM](https://github.com/hlxsites/merative2/assets/1815714/73f5302c-124f-4b30-a451-93b800600717)